### PR TITLE
Add read only prop to hoc-input

### PR DESCRIPTION
**Pull request by fgameiro**
closes: #243

### CODE REVIEW

File: packages/input-select/src/Select.story.js [hunk 1]
```diff
@@ -95,4 +95,18 @@ storiesOf("Select Input", "module")
...
+  .add(
+    "Read Only",
+    withInfo()(() => (
+      <Wrapper>
+        <Select
+          placeholder="Select fruit"
+          label="Fruit"
+          items={items}
+          onChange={action("onChange")}
+          readOnly
```
Comment (alcferreira) [inline packages/input-select/src/Select.story.js]: `readOnly` gives me the idea that the Select is disabled. The goal of this property is to disable the search feature.

Our suggestion is to receive a property called `disableAutoComplete` or something like that and disable the formInput based on this property.

Comment (alcferreira) [reply packages/input-select/src/Select.story.js]: Or even `disableSearch`

File: packages/input-select/src/Select.js [hunk 2]
```diff
@@ -157,6 +164,7 @@ Select.propTypes = {
   onChange: PropTypes.func,
   renderItem: PropTypes.func,
   noResultsMessage: PropTypes.string,
+  disableAutocomplete: PropTypes.bool,
```
Comment (fczuardi) [inline packages/input-select/src/Select.js]: I dont know If I like this name, this property disables typing in the input

**Comment by fgameiro** [comment]
What do you guys think of `disableTyping`? `disableEdit`?
I actually like both better... that propert prevents the typing, which consequently avoids the autocompletion.

**Comment by alcferreira** [comment]
I like `disableSearch` better, but it's up to you guys

**Comment by fgameiro** [comment]
`disabledSearch` then :)

**Pull request merged**